src/routes/tutorial/[slug]/codemirror.css

```diff
@@ -23,6 +19,10 @@
 	width: 5rem;
 }
 
+.cm-editor .cm-activeLine {
+	background: var(--sk-back-translucent);
+}
+
```

Comment:
When dark mode in OS and light mode in learn.svelte.dev (and vice versa), the background of the active line looks like:

<img width="971" alt="image" src="https://user-images.githubusercontent.com/29677552/232073821-6c26a3c1-cda9-4030-9d0b-53706da609b5.png">

---

Adding the above code, it will only depend on the site's scheme, and will look like this:

<img width="485" alt="image" src="https://user-images.githubusercontent.com/29677552/232074743-0df7514a-1220-4b8f-be28-75b48f0d426e.png">